AnnoDesigner.Core/Layout/Helper/StatisticsCalculationHelper.cs

```diff
@@ -19,16 +19,52 @@ public StatisticsCalculationResult CalculateStatistics(List<AnnoObject> placedOb
 
             var result = new StatisticsCalculationResult();
 
-            if (placedObjects.Count == 0)
+            if (!placedObjects.Any())
             {
                 return result;
             }
 
+            #region old logic is easier to understand, but slower
```

Comment:
Probably just use a comment for this, rather than a #region.

Reply:
change it to a comment